mypy/test/teststubgen.py

```diff
@@ -64,6 +64,23 @@ def test_packages_found(self) -> None:
             finally:
                 os.chdir(current)
 
+    def test_module_not_found(self) -> None:
+        current = os.getcwd()
+        with tempfile.TemporaryDirectory() as tmp:
+            try:
+                import io
+                capturedOutput = io.StringIO()
+                sys.stdout = capturedOutput
+                os.chdir(tmp)
+                self.make_file(tmp, 'mymodule.py', content='import a')
+                opts = parse_options(['-m', 'mymodule'])
+                py_mods, c_mods = collect_build_targets(opts, mypy_options(opts))
+                sys.stdout = sys.__stdout__
+                match = re.search('No module named \'a\'', capturedOutput.getvalue())
+                self.assertTrue(match)
```

Comment:
You can probably simply use `self.assertRegex()` instead of the two above lines.